clomonitor-core/src/linter/checks/trademark_disclaimer.rs

```diff
@@ -54,5 +55,8 @@ mod tests {
         assert!(TRADEMARK_DISCLAIMER.is_match(
             "The Linux Foundation® (TLF) has registered trademarks and uses trademarks."
         ));
+        assert!(
+            TRADEMARK_DISCLAIMER.is_match("Copyright © Kubernetes a Series of LF Projects, LLC")
```

Comment:
Looks like this hard codes "Kubernetes" instead of the wildcard for the project name?

Reply:
Ah, thank you